src/OWML.ModHelper.Menus/CustomInputs/OWMLTextEntryElement.cs

```diff
@@ -23,9 +23,14 @@ public string GetInputText()
 			return _popup.GetInputText();
 		}
 
-		public void SetCurrentValue(string text)
```

Comment:
Do we know if any mods depend on this method? Would this break them?

Reply:
Dont think so, but changed back to be save.